### PR TITLE
Add buttons

### PR DESCRIPTION
**Pull request by golubs**

### CODE REVIEW

File: client/src/components/Buttons/AddNewNoteButton.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Icon = styled.img``;
```
Comment (lmachens) [inline client/src/components/Buttons/AddNewNoteButton.js]: I would avoid having empty styled components.

File: client/src/components/Buttons/StartRepetitionButton.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import LogoIcon from '../Logo/LogoIcon';
+
+const Button = styled.button`
+  background: ${props => props.theme.colors.primary};
+  height: 70px;
+  width: 134px;
+
+  color: ${props => props.theme.colors.whitefont};
```
Comment (lmachens) [inline client/src/components/Buttons/StartRepetitionButton.js]: you wouldn't name a color `white`. Instead, you try to find a semantic name for it.
Imagine that you have a dark mode and want to invert the colors here.

Comment (panchepanevski) [reply client/src/components/Buttons/StartRepetitionButton.js]: Maybe only `font` or `text` would be a good option.

File: client/src/components/Buttons/StartRepetitionButton.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  font-size: 2.1rem;
```
Comment (panchepanevski) [inline client/src/components/Buttons/StartRepetitionButton.js]: You have 2 `font-size`.

Comment (golubs) [reply client/src/components/Buttons/StartRepetitionButton.js]: Thanks!

**Pull request merged**